test/controllers/api/v2/hosts_controller_test.rb

```diff
@@ -259,6 +266,12 @@ def expect_attribute_modifier(modifier_class, args)
     assert_response :success
   end
 
+  test "should update hostgroup_id of host" do
```

Comment:
@dLobatog sure, please point me if am messing something somewhere, the other 36 tests mentioned, are they failing because of this change? am unsure about it.